src/Robo.php

```diff
@@ -139,7 +139,7 @@ public static function loadConfiguration($paths, $config = null)
      * @param null|\Consolidation\Config\ConfigInterface $config
      * @param null|\Composer\Autoload\ClassLoader $classLoader
      *
-     * @return \Psr\Container\ContainerInterface
+     * @return \League\Container\Container|\Psr\Container\ContainerInterface
```

Comment:
\League\Container\Container **isa** \Psr\Container\ContainerInterface, so it seems unnecessary to explicitly list it as a return type.